### ISSUE TITLE
Breadcrumbs "active" class doesn't advance

### ISSUE THREAD

**Issue by cocodedesigns** [Desertsnowman/Caldera-Forms#385]
I have a multi-page form with styling on the list.  I have a different style on the "active" list item, but as I progress through the form, it doesn't seem to advance the "active" class to the next item in the list.  It doesn't seem to affect the change of pages, just the breadcrumbs.

**Comment by Desertsnowman** [comment]
yes. I did a rather amateur way of doing multi page. I regret it to a point.
I am planning on redesigning the way I do multipage forms. More in the line of how I do conditionals. ( template based )

But yes, this needs to be refined. thanks.
